gateway-sp-comms/src/sp_response_ext.rs

```diff
@@ -255,12 +256,13 @@ impl SpResponseExt for SpResponse {
         }
     }
 
-    fn expect_sp_state(self) -> Result<SpState> {
+    fn expect_sp_state(self) -> Result<VersionedSpState> {
         match self {
-            Self::SpState(state) => Ok(state),
+            Self::SpState(state) => Ok(VersionedSpState::V1(state)),
```

Comment:
Do you want to rename `SpState` -> `SpStateV1`? Hubpack doesn't care about the names so that wouldn't affect serialization, but might be slightly nicer to look at? Similarly for the names of the old structs/enums that now have `...V2` successors.

Reply:
(I second this motion)

Reply:
Done in https://github.com/oxidecomputer/management-gateway-service/pull/92/commits/1879755cbe398073368338d11717a8ac8dc7fd93